web/tests/functional/comment/test_comment.py

```diff
@@ -275,3 +275,38 @@ def test_same_bug_hash(self):
         comments = self._cc_client.getComments(bug_new.reportId)
         user_comments, _ = separate_comments(comments)
         self.assertEqual(len(user_comments), 0)
+
+    def test_user_comments(self):
```

Comment:
This test case depends from the `test_comment` test above (this is the reason why there is a system comment in this test case), and if we rename the above mentioned function the test order may change. A better approach would be to move your checking after this line:
https://github.com/Ericsson/codechecker/blob/aefe6a68d35da56bc7e0a534d23aa53729a703b8/web/tests/functional/comment/test_comment.py#L183
We can do something similar to this and simplify the whole thing:
```py
details = self._cc_client.getReportDetails(bug.reportId)
user_c, system_c = separate_comments(details.comments)

self.assertEqual(len(user_c), len(user_comments))
self.assertEqual(user_c[0].message, user_comments[0].message)

self.assertEqual(len(system_c), len(system_comments))
self.assertEqual(system_c[0].message, system_comments[0].message)
```

Reply:
Alright I have made the required changes 👍🏻